src/main/java/org/folio/rest/core/model/RequestContext.java

```diff
@@ -0,0 +1,38 @@
+package org.folio.rest.core.model;
+
+import io.vertx.core.Context;
+import org.folio.rest.persist.DBClient;
+
+import java.util.Collections;
+import java.util.Map;
+
+public class RequestContext {
```

Comment:
This class is not used anywhere except in the DBClient constructor, which is also not called anywhere. So it can be deleted.